### DISCUSSION TITLE
Liberatube v1.7

### DISCUSSION THREAD

**Discussion by GoldDominik893** [GoldDominik893/liberatube#7] [General]
V1.7 gonna be released 25th march

**Comment by therandomspoon** [comment]
Don’t care
![Uploading 110637645cbf9f9857fa2d7067dfc578.png…]()

**Reply by GoldDominik893** [reply]
Hello

**Comment by GoldDominik893** [comment]
it will be delayed im ultra busy

**Reply by therandomspoon** [reply]
Don’t care ratio 
![Uploading 2b9ecfc97620739e685fefdf3cdcae1d.png…]()
